proto/viam/app/data/v1/data.proto

```diff
@@ -36,72 +36,76 @@ message Filters {
   string part_id = 9;
   string location_id = 10;
   string org_id = 11;
-  CaptureInterval interval = 12;
+  string mime_type = 12;
+  CaptureInterval interval = 13;
+}
+
+message CaptureMetadata {
+  string location_id = 1;
+  string robot_name = 2;
+  string robot_id = 3;
+  string part_name = 4;
+  string part_id = 5;
+  string component_type = 6;
+  string component_model = 7;
+  string component_name = 8;
+  string method_name = 9;
+  map<string, google.protobuf.Any> method_parameters = 10;
+  repeated string tags = 11;
+  google.protobuf.Timestamp time_requested = 12;
+  google.protobuf.Timestamp time_received = 13;
+  string mime_type = 14;
+  string file_id = 15;
```

Comment:
File id is actually generated server side, so the client wouldn't be able to do this.

I was wrong about what the id is here when you asked, I _think_ it should actually be the session_id. @agreenb if you can confirm?

Reply:
Yeah +1 on this, not too sure what this value is supposed to be, originally thought mongo id

Reply:
Hmm I think this should be the same `file_id` that would be generated in the `BinaryDataByFilterResponse` so that it could be used in the IDs lookup. This would only be filled in for binary data. 

I'm a bit confused by what you mean that the client couldn't provide this?

fwiw I think we should change `session_id` in data_sync.proto to just be `id` for now